src/main/java/roomescape/repository/ReservationRepository.java

```diff
@@ -0,0 +1,34 @@
+package roomescape.repository;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import roomescape.dao.ReservationDao;
+import roomescape.entity.Reservation;
+
+import java.util.List;
+
+@Repository
+public class ReservationRepository {
+
+    private final ReservationDao reservationDao;
```

Comment:
보통 dao 의 네이밍은 
```suggestion
    private final ReservationJdbcTemplateDao or MysqlReservationDao 
```
와 같이 어떤 클래스를 사용했는지를 명확하게 적어주는 편이 좋은 것 같더라고요

repository 와 dao 가 나온 이유도 이와 조금 더 맞을 것 같은데요
repository -> db 종류에 구애받지 않는다
dao -> db 종류에 구애받는다
라고 봤을 때 dao 의 느낌상 하나의 db or 라이브러리에 무조건 종속되다보니 네이밍에서부터 드러내주는 것이 좋은 것 같아요